libs/core/src/lib/fundamental-ngx-core.module.ts

```diff
@@ -89,6 +89,7 @@ import { DynamicPageModule } from './dynamic-page/dynamic-page.module';
 import { StatusIndicatorModule } from './status-indicator/status-indicator.module';
 import { GridListModule } from './grid-list/grid-list.module';
 import { FacetModule } from './facets/facet.module';
+import { UserActionsMenuModule } from './shellbar/public_api';
```

Comment:
can this refer to the [.module instead of public_api](https://github.com/SAP/fundamental-ngx/pull/4502#pullrequestreview-578270438)?